packages/lesswrong/lib/collections/chapters/callbacks.js

```diff
@@ -30,3 +32,38 @@ function ChaptersEditHTMLSerializeCallback (modifier, chapter) {
 }
 
 addCallback("chapters.edit.sync", ChaptersEditHTMLSerializeCallback);
+
+async function ChaptersEditCanonizeCallback (chapter) {
+  const posts = await Sequences.getAllPosts(chapter.sequenceId)
+  const sequence = await Sequences.findOne({_id:chapter.sequenceId})
+
+  _.range(posts.length).forEach((i) => {
```

Comment:
.map seems better here, and also comes with an index variable in the second argument